datasets/makhzan/dataset_infos.json

```diff
@@ -0,0 +1 @@
ed/makhzan/master/text/1436.xml": {"num_bytes": 5965, "checksum": "1e023f70c768dc69354c64779bd99206ea4d52b5e511ee1fdfb7ab27372890db"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1437.xml": {"num_bytes": 1433, "checksum": "277b094dd29b6f1634ceb47c1637a3aa05458d868004a488891b528450722a2d"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1438.xml": {"num_bytes": 2664, "checksum": "85fccfa36b410aa96d60610b4d52a9b97bc7afbf225d13b7a8f66bfd77d9135e"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1439.xml": {"num_bytes": 2730, "checksum": "c21c9fbca0b6e128f504dc0522a9b8146f7cd46279d57d948507e2e9f9b15705"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1440.xml": {"num_bytes": 12526, "checksum": "054c3f5e2904d633eb470878a06e1cad57bde5126e188a3c304d5b645de07083"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1441.xml": {"num_bytes": 12712, "checksum": "a4b0a1e12eff43a2f909fa4ca746f0b873e8c33cb65622c7fb266b704105d889"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1442.xml": {"num_bytes": 10463, "checksum": "d6765b7f46ad70cfd5c9e6a9aa2ca4f070621aca57f09fdf2991ba957d2d6da2"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1443.xml": {"num_bytes": 11103, "checksum": "67f291aebb863fa466d38c712651c8367d1357576a9071592bf2b1765e3696de"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1444.xml": {"num_bytes": 7726, "checksum": "662b74a8a1f9cb28ab9781de1b772ae925333f8a505daac560dbf274636e7f18"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1445.xml": {"num_bytes": 12092, "checksum": "5e8cf57247698b3785c0ab9a718b15442d9dd1d351aad52261c626826137beea"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1446.xml": {"num_bytes": 19258, "checksum": "5c7503a833f5f1a59160ee383f810db5ef623ece398785a10a18e6b2ba1742f0"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1447.xml": {"num_bytes": 8633, "checksum": "182113392f0ee99cea620493496602455b392dc0ec97f6b8d949f137a148c7d2"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1448.xml": {"num_bytes": 10539, "checksum": "b920c2300a0f042a33f641c9a09fcbd2e6d7798b26f2c6fc441b142bfdfc9e4f"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1449.xml": {"num_bytes": 4046, "checksum": "2c745be0d31aa1e608d44455ea8d68c65a0beaa51231b69c0deba20ce7d078c2"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1450.xml": {"num_bytes": 5961, "checksum": "9a3def9b182bc56385abb23d01b9f65bcc6872d5c952e1826dfa9ac9a6e69980"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1451.xml": {"num_bytes": 3635, "checksum": "2ebf58f52c90f37df7abbe22e7c1c84ac7a4eb52834d6ce639bfcc45361ce6ff"}, "https://raw.githubusercontent.com/zeerakahmed/makhzan/master/text/1452.xml": {"num_bytes": 1149, "checksum": "0eb21670f38e8fc92498f54563cf9d9bd56e5fb9a68c16905b668716776130
```

Comment:
looks like the dataset_infos.json contain some old informations about the config that was named `train`.

Could you delete dataset_infos.json and regenerate it please ?
You must regenerate it after the URL change